packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -144,6 +148,22 @@ const isWithinAvailableHours = (
       return true;
     }
   }
+
+  // check if it is a date override
+  for (const dateOverride of dateOverrides) {
+    const utcOffSet = dayjs(dateOverride.start).tz(inviteeTimeZone).utcOffset();
```

Comment:
<img width="851" alt="Screenshot 2023-04-26 at 7 50 20 PM" src="https://user-images.githubusercontent.com/1780212/234605298-ae881df6-96c4-4990-90c5-a2b80bfbd7d8.png">
As per my understanding, dateOverride.start doesn't matter. The utcOffset depends just on inviteeTimeZone.
So, we can make it and then move outside the loop and compute just once.
```javascript
    const utcOffSet = dayjs().tz(inviteeTimeZone).utcOffset();
```

Reply:
Thanks that makes sense 🙏